btree/btree.c

```diff
@@ -1501,6 +1502,253 @@ static void generic_move(struct nd *src, struct nd *tgt,
 	node_fix(tgt, tx);
 }
 
+
+/**
+ *  -------------------------------------------------------
+ *  Section START - Credit calculation for BTree Operations
+ *  -------------------------------------------------------
+ */
+
+//In case of fix key, fix value node format: max_ksize, max_vsize will be fix key, value size else estimate max sizes
+m0_bcount_t max_ksize; 
+m0_bcount_t max_vsize;
+m0_bcount_t max_node_size;
+m0_bcount_t internal_vsize; //for internal node, value size is fix
+
+/** threshold can be used in btree_move_key_value_credit() to identify 
+ * what fraction of node needs to be move in case of overflow/underflow
+ */
+enum threshold{
```

Comment:
```
enum threshold {
```
a space is needed after the identifier.

Reply:
What threshold?
Please have a more meaningful name, and add comments.

Reply:
In some conditions, such as underflow/overflow, there will be some threshold point, after that we will move records between two nodes. To figure out how many records or fraction of node needs to be moved, we can use threshold . e.g, in case of deletion, there can be underflow, so in worst case , we might need to move ~50% of records(in case of merging), therefore, MIN_MOVE will be used, and later on we can update these values, once we decide the conditions for underflow, overflow